### ISSUE TITLE
LoadProject is called twice on solution load due to configuration change

### ISSUE THREAD

**Issue by KirillOsenkov** [mono/monodevelop#4632]
1. Debug MonoDevelop, place breakpoint in MonoDevelopWorkspace.LoadProject()
2. Open simple C# solution with one C# console app

Expected: LoadProject is called once

Actual: LoadProject is called the second time when the active configuration changes:

```
>	MonoDevelop.Ide.dll!MonoDevelop.Ide.TypeSystem.MonoDevelopWorkspace.CreateSolutionInfo Line 214	C#
	MonoDevelop.Ide.dll!MonoDevelop.Ide.TypeSystem.MonoDevelopWorkspace.HandleActiveConfigurationChanged Line 198	C#
 	MonoDevelop.Ide.dll!MonoDevelop.Ide.RootWorkspace.OnActiveConfigurationChanged Line 111	C#
 	MonoDevelop.Ide.dll!MonoDevelop.Ide.RootWorkspace.RestoreWorkspacePreferences Line 695	C#
 	MonoDevelop.Ide.dll!MonoDevelop.Ide.RootWorkspace.BackgroundLoadWorkspace Line 632	C#
```

> VS bug [#603762](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/603762)

**Comment by mkrueger** [comment]
@lluis: I already saw that however the process gets canceled and solution creation is not that a big issue. 

But in general it would be nice if a project would be loaded with everything set up already including configuration. That would really help.

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-18] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #4891 merged by: Lluis Sanchez  
 Author: mono  
 Commit: b1a969312ca32a72a887b1e1382940dd3fefb8d4 (xamarin/monodevelop)
